### PR TITLE
YARN-10443: Document options of logs CLI

### PR DESCRIPTION
**Pull request by ankit-kumar-25**
What? :: Document options of logs CLI

https://issues.apache.org/jira/browse/YARN-10443

@adamantal Can you please review this?

Thanks!

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  9s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  31m 43s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 24s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  47m 52s |  branch has no errors when building and testing our client artifacts.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 11s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 16s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  15m 32s |  patch has no errors when building and testing our client artifacts.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 29s |  The patch does not generate ASF License warnings.  |
|  |   |  67m 14s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2325/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2325 |
| JIRA Issue | YARN-10443 |
| Optional Tests | dupname asflicense mvnsite markdownlint |
| uname | Linux 7291926c8f02 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 6b5d9e2334b |
| Max. process+thread count | 308 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site U: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2325/1/console |
| versions | git=2.17.1 maven=3.6.0 |
| Powered by | Apache Yetus 0.13.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

File: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/YarnCommands.md [hunk 1]
```diff
@@ -122,8 +122,20 @@ Usage: `yarn logs -applicationId <application ID> [options] `
...
+| -client_retry_interval_ms \<Retry Interval\> | Retry Interval (work with client_max_retries) |
+| -clusterId \<Cluster ID\> | ClusterId |
+| -list_nodes | Show the list of nodes that successfully aggregated logs (can only be used with finished applications) |
+| -log_files \<Log File Name\> | View a specific log type |
+| -log_files_pattern \<Log File Pattern\> | Get matched log files by using java regex |
+| -out \<Local Directory\> | Download logs to specified local folder |
+| -show_application_log_info | List all container IDs (combine this with --nodeAddress to get containerIds for all the containers on the specific NodeManager) |
+| -show_container_log_info | Show the container log metadata |
+| -size \<size\> | To list only a portion of the log files for a particular container |
+| -size_limit_mb \<Size Limit\> | Size of the total logs which could be fetched |
```
Comment (adamantal) [inline hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/YarnCommands.md:137]: Could you please rephrase these two sentences?
Both options limits the size of logs that is retrieved, but it is not mentioned that `-size` limits the number of bytes, and the `-size_limit_mb` limits the megabytes.

**Review by adamantal — CHANGES_REQUESTED** [review]
Hi @ankit-kumar-25, thanks for working on this. Looks good overall, had two comments.

Could you please reorganize the options and put them into alphabetical order (including the existing options)?

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |  32m 40s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  markdownlint  |   0m  0s |  |  markdownlint was not available.  |
| +1 :green_heart: |  @author  |   0m  1s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  33m  0s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 24s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  49m 27s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 12s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 15s |  |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  16m 37s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 29s |  |  The patch does not generate ASF License warnings.  |
|  |   | 101m 26s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2325/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2325 |
| JIRA Issue | YARN-10443 |
| Optional Tests | dupname asflicense mvnsite markdownlint |
| uname | Linux 8e21284f6d14 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 18f9d77c46f |
| Max. process+thread count | 345 (vs. ulimit of 5500) |
| modules | C: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site U: hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2325/2/console |
| versions | git=2.17.1 maven=3.6.0 |
| Powered by | Apache Yetus 0.13.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

**Comment by adamantal** [comment]
Thanks for working on this @ankit-kumar-25. Pushed to trunk.

**Pull request closed without merging**